packages/js/e2e-environment/config/jest-object.config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External Dependencies
+ */
+const { E2E_RETRY_TIMES } = process.env;
+
+const setupJestRetries = () => {
```

Comment:
Since we are exporting this function for use in other projects we should add a `retries = 0` parameter. Then 

```
const retryTimes = E2E_RETRY_TIMES ? E2E_RETRY_TIMES : retries;

if ( retryTimes ) {
jest.retryTimes( retryTimes );
}
```

That will allow a project to be set up with a default number of retries and have it overridden by the constant.

Reply:
That's a great suggestion. I've made the changes